<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>360</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QListWidge操作</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>121</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QToolButton" name="toolButton_5">
          <property name="text">
           <string>删除项</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_2">
          <property name="text">
           <string>清除列表</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>列表初始化</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_3">
          <property name="text">
           <string>插入项</string>
          </property>
         </widget>
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>添加项</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>360</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QTreeWidget操作</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>360</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QTableWidget操作</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>661</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>QLIstWidget</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>78</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>127</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>421</width>
        <height>30</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolButton" name="toolButton_6">
       <property name="text">
        <string>选择项目</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_7">
       <property name="text">
        <string>选择所有项目</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_8">
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_9">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>QTreeWidget</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>QTableWidget</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
